bin/setup.sh

```diff
@@ -190,7 +190,7 @@ autolab_setup() {
     cp $AUTOLAB_PATH/config/school.yml.template $AUTOLAB_PATH/config/school.yml
 
     cp $AUTOLAB_PATH/config/initializers/devise.rb.template $AUTOLAB_PATH/config/initializers/devise.rb
-    sed -i "s/<YOUR-SECRET-KEY>/`bundle exec rake secret`/g" $AUTOLAB_PATH/config/initializers/devise.rb
+    sed -i "s/<YOUR-SECRET-KEY>/`sh generate_secret.sh 128`/g" $AUTOLAB_PATH/config/initializers/devise.rb
```

Comment:
FYI, on Mac I think the proper command is `sed -i ".bak" "s/<YOUR-SECRET-KEY>/`bash generate_secret.sh 128`/g" target/file`, i.e. we must provide a suffix (".bak") when running `sed -i`. 

I ran into this issue while following the installation guide: https://autolab.github.io/docs/